test/explorer/backend/lazy_series_test.exs

```diff
@@ -5,7 +5,7 @@ defmodule Explorer.Backend.LazySeriesTest do
   alias Explorer.Backend.LazySeries
 
   test "inspect/2 gives a basic hint of lazy series" do
-    data = LazySeries.new(:column, ["col_a"])
+    data = LazySeries.new(:column, ["col_a"], :unknown)
```

Comment:
In a few tests like this, I added an `:unknown` dtype. This was the only file I found where we were making a `LazySeries` and didn't also know the `dtype` when we did so.